stdlib/public/SDK/Metal/Metal.swift

```diff
@@ -100,10 +100,11 @@ extension MTLDevice {
 @available(swift 4)
 @available(macOS 10.13, *)
 public func MTLCopyAllDevicesWithObserver(handler: @escaping MTLDeviceNotificationHandler) -> (devices:[MTLDevice], observer:NSObject) {
-    var resultTuple: (devices:[MTLDevice], observer:NSObject)
-    resultTuple.observer = NSObject()
-    resultTuple.devices = __MTLCopyAllDevicesWithObserver(AutoreleasingUnsafeMutablePointer<NSObjectProtocol?>(&resultTuple.observer), handler)
-    return resultTuple
+    var observer: NSObjectProtocol?
+    let devices = __MTLCopyAllDevicesWithObserver(&observer, handler)
+    // FIX-ME: The force cast here isn't great – ideally we would return the
```

Comment:
Nitpick: the `-` in fixme makes it undiscoverable. All other `FIXME`'s in the codebase don't have it.

Reply:
Good spot, thanks!